coins.json

```diff
@@ -1065,5 +1065,22 @@
             "clientPublic": "",
             "clientDocs": "https://fio.foundation"
         }
+    },
+    {
+        "id": "solana",
+        "name": "Solana",
+        "symbol": "SOL",
+        "decimals": 13,
+        "blockchain": "Solana",
+        "derivationPath": "m/44'/501'/0'",
+        "curve": "ed25519",
+        "publicKeyType": "ed25519",
+        "explorer": "https://explorer.solana.com/tx/",
+        "info": {
+            "url": "https://solana.com",
+            "client": "https://solana.com",
+            "clientPublic": "",
```

Comment:
Do you have public nodes?

Reply:
https://developer.trustwallet.com/wallet-core/contributing#coin-definition

For Ethereum:
client: https://github.com/ethereum/go-ethereum
clientPublic: https://mainnet.infura.io